src/ProjectSystemSetup/ProjectSelectors.pkgdef

```diff
@@ -2,3 +2,7 @@
 "ProjectSelector"="{3DCC42E7-4611-46AC-8EA8-6F1E9B3D917F}"
 [$RootKey$\Projects\{F184B08F-C81C-45F6-A57F-5ABD9991F28F}]
 "ProjectSelector"="{50018B05-93AB-437C-A9EB-F4AD12FF83DD}"
+[$RootKey$\Projects\{F2A71F9B-5D33-465A-A702-920D77279786}]
+"ProjectSelector"="{E720DAD0-1854-47FC-93AF-E719B54B84E6}"
+[$RootKey$\ProjectSelectors\{E720DAD0-1854-47FC-93AF-E719B54B84E6}]
+"Package"="{a724c878-e8fd-4feb-b537-60baba7eda83}"
```

Comment:
is this GUID supposed to be lowercase (it probably doesn't matter, but the others are all upper)?